content/en/internal/configuration-database/_index.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: "Configuration Database"
-description: "Service configuration database"
 weight: 20
 type: "docs"
+description: >
+    Topology and configuration registry for sites in EGI FedCloud
```

Comment:
That's not correct, it's not at all limited to EGI FedCloud, it's for the whole infrastructure.

Reply:
Fixed